cmd/homi/setup/cmd.go

```diff
@@ -478,28 +478,37 @@ func gen(ctx *cli.Context) error {
 	privKeys, nodeKeys, nodeAddrs := istcommon.GenerateKeys(cnNum)
 	testPrivKeys, testKeys, testAddrs := istcommon.GenerateKeys(numTestAccs)
 
-	var genesisJsonBytes []byte
+	var (
+		genesisJson      *blockchain.Genesis
+		genesisJsonBytes []byte
+	)
 
 	validatorNodeAddrs := make([]common.Address, numValidators)
 	copy(validatorNodeAddrs, nodeAddrs[:numValidators])
 
 	if cypressTest {
-		genesisJsonBytes, _ = json.MarshalIndent(genCypressTestGenesis(validatorNodeAddrs, testAddrs), "", "    ")
+		genesisJson = genCypressTestGenesis(validatorNodeAddrs, testAddrs)
 	} else if cypress {
-		genesisJsonBytes, _ = json.MarshalIndent(genCypressGenesis(validatorNodeAddrs, testAddrs), "", "    ")
+		genesisJson = genCypressGenesis(validatorNodeAddrs, testAddrs)
 	} else if baobabTest {
-		genesisJsonBytes, _ = json.MarshalIndent(genBaobabTestGenesis(validatorNodeAddrs, testAddrs), "", "    ")
+		genesisJson = genBaobabTestGenesis(validatorNodeAddrs, testAddrs)
 	} else if baobab {
-		genesisJsonBytes, _ = json.MarshalIndent(genBaobabGenesis(validatorNodeAddrs, testAddrs), "", "    ")
+		genesisJson = genBaobabGenesis(validatorNodeAddrs, testAddrs)
 	} else if cliqueFlag {
-		genesisJsonBytes, _ = json.MarshalIndent(genCliqueGenesis(ctx, validatorNodeAddrs, testAddrs, chainid), "", "    ")
+		genesisJson = genCliqueGenesis(ctx, validatorNodeAddrs, testAddrs, chainid)
 	} else if serviceChain {
-		genesisJsonBytes, _ = json.MarshalIndent(genServiceChainGenesis(validatorNodeAddrs, testAddrs), "", "    ")
+		genesisJson = genServiceChainGenesis(validatorNodeAddrs, testAddrs)
 	} else if serviceChainTest {
-		genesisJsonBytes, _ = json.MarshalIndent(genServiceChainTestGenesis(validatorNodeAddrs, testAddrs), "", "    ")
+		genesisJson = genServiceChainTestGenesis(validatorNodeAddrs, testAddrs)
 	} else {
-		genesisJsonBytes, _ = json.MarshalIndent(genIstanbulGenesis(ctx, validatorNodeAddrs, testAddrs, chainid), "", "    ")
+		genesisJson = genIstanbulGenesis(ctx, validatorNodeAddrs, testAddrs, chainid)
 	}
+
+	genesisJson.Config.IstanbulCompatibleBlock = big.NewInt(0)
```

Comment:
@yoomee1313 Do we need to add a flag for it?

Reply:
@jeongkyun-oh Since homi is a tool for creating a new network, the option is deliberately omitted to induce `istanbulCompatibleBlock` to be used by default.

Reply:
It is a good idea to add `istanbulCompatible block number` flag. The flag is applied it at https://github.com/klaytn/klaytn/pull/1070/commits/081644a7c90ee260b34703bf22b1faad4de5f9c2. Thanks!